### PR TITLE
9345 scaffold pcpg

### PR DESCRIPTION
**Pull request by cohnjesse**
## Description
[Original Ticket](https://github.com/department-of-veterans-affairs/va.gov-team/issues/7224)

This ticket is to scaffold out the basic folder and file structure for the PCPG form

## Acceptance criteria
- [x] The initial pages have been created locally and a quick view has been shared with the team

### CODE REVIEW

File: src/applications/planning-guidance/8832/28-8832-schema.json [hunk 1]
```diff
@@ -0,0 +1,216 @@
...
+  "title": "28-8832 Personal career planning and guidance (28-8832)",
+  "type": "object",
+  "additionalProperties": false,
+  "definitions": {
+    "address": {
+      "type": "object",
+      "oneOf": [
+        {
+          "properties": {
+            "country": {
```
Comment (micahchiang) [inline src/applications/planning-guidance/8832/28-8832-schema.json:12]: Where does this schema come from? Will address always be limited to the USA, Canada, and Mexico?

File: src/applications/planning-guidance/8832/28-8832-schema.json [hunk 2]
```diff
@@ -0,0 +1,216 @@
...
+      ],
+      "properties": {
+        "street": {
+          "type": "string",
+          "minLength": 1,
+          "maxLength": 50
+        },
+        "street2": {
+          "type": "string",
+          "minLength": 1,
```
Comment (micahchiang) [inline src/applications/planning-guidance/8832/28-8832-schema.json:205]: Is street 2 usually optional? Will requiring a min length throw a validation error if the field is left blank?

File: src/applications/planning-guidance/8832/config/form.js [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+          uiSchema: veteranInformation.uiSchema,
+          schema: veteranInformation.schema,
+        },
+        veteranAddress: {
+          path: 'veteran-address',
+          title: 'Veteran Address',
+          uiSchema: veteranAddress.uiSchema,
+          schema: veteranAddress.schema,
+        },
+        vereranOptional: {
```
Comment (micahchiang) [inline src/applications/planning-guidance/8832/config/form.js:53]: This should be veteranOptional

File: src/applications/planning-guidance/8832/config/pages/01-selection.js [hunk 4]
```diff
@@ -0,0 +1,21 @@
+export const schema = {
+  type: 'object',
+  properties: {
+    applicantType: {
+      type: 'string',
+      title: 'Select one of the option below',
```
Comment (micahchiang) [inline src/applications/planning-guidance/8832/config/pages/01-selection.js:6]: This should be options

File: src/applications/planning-guidance/8832/helpers.js [hunk 5]
```diff
@@ -0,0 +1,7 @@
+import React from 'react';
+
+export const directDepositWarning = (
+  <div className="pension-dd-warning">
+    The Department of Treasury requires all federal benefit payments be made by electronic funds transfer (EFT), also called direct deposit. If you don’t have a bank account, you must get your payment through Direct Express Debit MasterCard. To request a Direct Express Debit MasterCard you must apply at <a href="http://www.usdirectexpress.com" target="_blank" rel="noopener noreferrer">www.usdirectexpress.com</a> or by telephone at <a href="tel:8003331795" target="_blank">1-800-333-1795</a>. If you chose not to enroll, you must contact representatives handling waiver requests for the Department of Treasury at <a href="tel:8882242950" target="_blank">1-888-224-2950</a>. They will address any questions or concerns you may have and encourage your participation in EFT.
```
Comment (micahchiang) [inline src/applications/planning-guidance/8832/helpers.js:5]: These phone numbers should have Aria labels I think

File: src/applications/planning-guidance/8832/manifest.json [hunk 6]
```diff
@@ -0,0 +1,7 @@
+{
```
Comment (micahchiang) [inline src/applications/planning-guidance/8832/manifest.json:1]: Can't recall right now, but do we need to add anything to registry.json for this application?

**Review by micahchiang — CHANGES_REQUESTED** [review]
Requesting changes for some name conventions. Also left some other questions.

File: src/applications/planning-guidance/8832/28-8832-schema.json (hunk 1 above)
Comment (cohnjesse) [reply src/applications/planning-guidance/8832/28-8832-schema.json:12]: All of the code in here is boilerplate code

File: src/applications/planning-guidance/8832/28-8832-schema.json (hunk 2 above)
Comment (cohnjesse) [reply src/applications/planning-guidance/8832/28-8832-schema.json:205]: this is all boilerplate code and will all change

File: src/applications/planning-guidance/8832/config/form.js (hunk 3 above)
Comment (cohnjesse) [reply src/applications/planning-guidance/8832/config/form.js:53]: fixed

File: src/applications/planning-guidance/8832/config/pages/01-selection.js (hunk 4 above)
Comment (cohnjesse) [reply src/applications/planning-guidance/8832/config/pages/01-selection.js:6]: this is all just placeholder code, none of it is permanent and will all change

File: src/applications/planning-guidance/8832/helpers.js (hunk 5 above)
Comment (cohnjesse) [reply src/applications/planning-guidance/8832/helpers.js]: this is boilderplate code and we won't be using this in our app. deleted.

File: src/applications/planning-guidance/8832/manifest.json (hunk 6 above)
Comment (cohnjesse) [reply src/applications/planning-guidance/8832/manifest.json:1]: We will at some point, this is just a ticket to get the folder and file structure set up, nothing more.

**Review by micahchiang — APPROVED** [review]

**Review by rianfowler — APPROVED** [review]
Recommend assigning this app folder to your team in the CODEOWNERS config.

**Pull request merged**